frontend/src/pages/Projects/Project/ProjectDetails/ClientCodingSection/index.tsx

```diff
@@ -6,7 +6,7 @@ import { useFormControls } from "hooks";
 import { useFormData } from "hooks/useFormData";
 import { IFormControls } from "types";
 import { tableConfig } from "./tableConfig";
-import { FormConfig } from "./FormConfig";
+import { FormConfig } from "./formConfig";
```

Comment:
This seems to be causing a fatal error when I pull the branch, if you can point out how I am in error, I'll be happy to approve.

Reply:
Suggest these changes: https://github.com/bcgov/gdx-agreements-tracker/tree/descw-1219-kshapka-suggested-changes